user/mods/Trap-PathToTarkov/config/config.json

```diff
@@ -394,7 +391,7 @@
     "factory4_night": {
       "Gate 3": "MechanicStash",
       "Office Window": "JaegerStash",
-      "Cellars": "AnastasiaHideout"
+      "Cellars": "AnastasiaStash"
```

Comment:
⚠️ AnastasiaHideout was the name of the offraid position, you need to change the different infils and `access_via` to match this offraid position.

Reply:
Already did, I just forget to change these lines
https://github.com/guillaumearm/SPT_The_Escape/blob/32ff0d5012f17b88d1f1d8759e8484246185da4a/user/mods/Trap-PathToTarkov/config/config.json#L502

Reply:
Oh you right, so it's a fix ;-)